src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionLikelihoods.java

```diff
@@ -143,11 +141,11 @@ protected static double logLikelihoodForAllelicPanelOfNormals(final double meanB
         return counts.getCounts().stream().mapToDouble(c -> AlleleFractionLikelihoods.logIntegralOverAllelicBias(alpha, beta, balancedMinorAlleleFraction, c.getAltReadCount(), c.getRefReadCount())).sum();
     }
 
-    private static double alpha(final double meanBias, final double biasVariance) {
+    public static double alpha(final double meanBias, final double biasVariance) {
```

Comment:
hmm, now that these are public, might want to add parameter checking (or add javadoc saying it will not be done).  will leave it up to you.